### DISCUSSION TITLE
clock centered on taskbar

### DISCUSSION THREAD

**Discussion by Torinde** [valinet/ExplorerPatcher#875] [Ideas]
Win11 taskbar has the option to move start/applications to the center. But the clock/tray is always on the far right. On ultrawide displays, especially large "Super Ultrawide", it would be handy to be able to choose the clock/tray position (center vs right) as well.

I prefer Win10 taskbar, because it provides the ">>" cascaded browsing from its desktop toolbar (#874) - but can you modify that as well to have start/applications/clock in the center?

**Comment by Poopooracoocoo** [comment]
Sounds like TaskbarX to me :/

**Reply by Torinde** [reply]
TaskbarX moves the applications/shortcuts to the center. But Start remains on the left and Clock remains on the right.
I would like everything on the taskbar to be centered (because the far left/right corners are TOO FAR on a very big monitor).

Ideally, the taskbar should not even occupy any space to the left of Start and to the right of Clock - providing more vertical space for desktop icons and application windows.

Mockup:
![image](https://user-images.githubusercontent.com/97228894/154808156-9bb49bab-2c06-43a9-9a43-523212870d78.png)

**Reply by Poopooracoocoo** [reply]
Oh ok

**Comment by Torinde** [comment]
Same at:
- ChrisAnd1998/TaskbarX#912
- https://github.com/valinet/ExplorerPatcher/issues/3398#issuecomment-2829942907
- https://github.com/ramensoftware/windhawk/discussions/487
